SayTheirNames/Source/Dependency/Deep Linking/Deep Links/AboutDeepLink.swift

```diff
@@ -25,20 +25,13 @@
 import UIKit
 
 struct AboutDeepLink: DeepLink {
-    let linkDetails: DeepLinkDetails
     static let details = DeepLinkDetails(
         displayClass: MoreController.self,
         type: AboutDeepLink.self
     )
-    .schemes(["stn", "https"])
-    .host("saytheirname.netlify.app")
-    .path("about")
+    .schemes(DeepLinkSupport.Schema.all)
+    .host(DeepLinkSupport.Host.default)
+    .path(DeepLinkSupport.Path.about)
     
-    init() {
-        self.linkDetails = type(of: self).details
-    }
-    
-    static func deepLink(fromComponents: [String]? = nil) -> DeepLink? {
-        return self.init()
-    }
+    init?(components: [String]?) { }
```

Comment:
if this initializer never fails, it doesn't need to be failable (regular inits still satisfy the requirement)

Reply:
Done